aws-mfa

```diff
@@ -276,6 +288,7 @@ def get_credentials(short_term_name, lt_key_id, lt_access_key, args, config):
     logger.info(
         "Success! Your credentials will expire in %s seconds at: %s"
         % (args.duration, response['Credentials']['Expiration']))
+    sys.exit(0)
```

Comment:
I was only able to replicate the following scenario:

1.  Successfully auth `aws-mfa --duration 900 --assume-role arn:aws:iam::666666666999:role/Admin --role-session-name test`
2. Manually modify a field in the short term credentials, for example, changing `aws_session_token` to `ZZZaws_session_token`
3. You receive the message: `WARNING - Your existing credentials are missing or invalid, obtaining new credentials.`
4. Re-auth `aws-mfa --duration 900 --assume-role arn:aws:iam::666666666999:role/Admin --role-session-name test`, then receive the duplicate error message.

While this seems like a pretty rare case, I'm going to go ahead and keep the exit.